### PR TITLE
add polkadot support

### PR DESCRIPTION
**Pull request by galvanizze**

### CODE REVIEW

File: blockapi/v2/api/subscan.py [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+            coin=self.coin,
+            asset_type=AssetType.REWARDS,
+            raw={},
+        )
+
+    def _yield_staking_rewards(self, address: str) -> Iterable[Decimal]:
+        body = {
+            'address': address,
+            'row': self.api_options.max_items_per_page,
+            'page': 0,
```
Comment (ladovina) [inline blockapi/v2/api/subscan.py:123]: question: Is pagination needed here? Why balances does not use it?

Comment (galvanizze) [reply blockapi/v2/api/subscan.py:123]: yes it's needed, rewards in polkadot are special events, 1 per era, so there is a loooot of items (~1k)

balances structure is single dict (flat), with totals, no need for pagination

**Review by ladovina — APPROVED** [review]

File: blockapi/v2/api/subscan.py [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+                asset_type=AssetType.AVAILABLE,
+                raw=data,
+            )
+
+        if b_staked:
+            yield BalanceItem.from_api(
+                balance_raw=int(b_staked),
+                coin=self.coin,
+                asset_type=AssetType.STAKED,
+                raw=data,
```
[the remark this answers is not in the document]
Comment (galvanizze) [reply blockapi/v2/api/subscan.py:88]: first I was not sure, if `data['balance_lock']` is total staked amount, with unbonding amount, so It was computed before..

and `b_staked` is used for calculation of available account, this is straightforward for me..

File: blockapi/v2/api/subscan.py [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+                asset_type=AssetType.VESTING,
+                raw=data['vesting'],
+            )
+
+        if b_locked:
+            yield BalanceItem.from_api(
+                balance_raw=int(b_locked),
+                coin=self.coin,
+                asset_type=AssetType.LOCKED,
+                raw=data,
```
[the remark this answers is not in the document]
Comment (galvanizze) [reply blockapi/v2/api/subscan.py:104]: it's more straightforward, and maybe we'll add `democracy` and `election` types here (I did it before, but really don't know what those fields mean 🤷 so I deleted it and leave comment)

**Pull request merged**